SYCL/Matrix/XMX8/element_wise_add_jm_apply_bf16.cpp

```diff
@@ -0,0 +1,25 @@
+//==----------- element_wise_all_ops_bf16.cpp  - DPC++ joint_matrix---------==//
```

Comment:
change name of test on line 1.
Also, change name of the file to: joint_matrix_apply_bf16.cpp

Reply:
Done